tests/integration/encryption/utils.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package encryption
+
+import (
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+const userCollectionGQLSchema = (`
+	type Users {
+		name: String
+		age: Int @crdt(type: "lww")
```

Comment:
question: Why the explicit `@crdt(type: "lww")` - is it required?

Reply:
Ah okay makes sense - can you please add a comment explain that, else it is likely to be removed or cause head scratching at a later date :)

Reply:
done